traffic_ops/testing/api/v4/deliveryservice_requests_test.go

```diff
@@ -16,672 +16,345 @@ package v4
+							"multiSiteOrigin":      false,
+							"orgServerFqdn":        "http://example.test",
+							"protocol":             0,
+							"qstringIgnore":        0,
+							"rangeRequestHandling": 0,
+							"regionalGeoBlocking":  true,
+							"routingName":          strings.Repeat("X", 1) + "." + strings.Repeat("X", 48),
+							"tenant":               "tenant1",
+							"type":                 "HTTP",
+							"xmlId":                "X " + strings.Repeat("X", 46),
+						},
+						"status": "draft",
+					},
+					Expectations: utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
+				},
+				"BAD REQUEST when NON-DRAFT": {
+					ClientSession: TOSession, RequestBody: map[string]interface{}{
+						"changeType": "create",
+						"requested": map[string]interface{}{
+							"active":               false,
+							"cdnName":              "cdn1",
+							"displayName":          "Testing transitions",
+							"dscp":                 3,
+							"geoLimit":             1,
+							"geoProvider":          1,
+							"initialDispersion":    1,
+							"ipv6RoutingEnabled":   true,
+							"logsEnabled":          true,
+							"missLat":              0.0,
+							"missLong":             0.0,
+							"multiSiteOrigin":      false,
+							"orgServerFqdn":        "http://example.test",
+							"protocol":             0,
+							"qstringIgnore":        0,
+							"rangeRequestHandling": 0,
+							"regionalGeoBlocking":  true,
+							"routingName":          "goodroute",
+							"tenant":               "tenant1",
+							"type":                 "HTTP",
+							"xmlId":                "test-transitions",
+						},
+						"status": "pending",
+					},
+					Expectations: utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
+				},
+				"BAD REQUEST when ALREADY EXISTS": {
+					ClientSession: TOSession, RequestBody: map[string]interface{}{
+						"changeType": "create",
+						"requested": map[string]interface{}{
+							"active":               true,
+							"cdnName":              "cdn1",
+							"displayName":          "Good Kabletown CDN",
+							"dscp":                 1,
+							"geoLimit":             1,
+							"geoProvider":          1,
+							"initialDispersion":    1,
+							"ipv6RoutingEnabled":   true,
+							"logsEnabled":          true,
+							"missLat":              0.0,
+							"missLong":             0.0,
+							"multiSiteOrigin":      false,
+							"orgServerFqdn":        "http://example.test",
+							"protocol":             0,
+							"qstringIgnore":        0,
+							"rangeRequestHandling": 0,
+							"regionalGeoBlocking":  true,
+							"routingName":          "goodroute",
+							"tenant":               "tenant1",
+							"type":                 "HTTP",
+							"xmlId":                "test-ds1",
+						},
+						"status": "draft",
+					},
+					Expectations: utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
+				},
+			},
+			"DELETE": {
```

Comment:
Why is there a case DELETE, when it is already covered in withobjs_test.go?

Reply:
See comment above